### PR TITLE
Fetch from canonical

### PR DESCRIPTION
**Pull request by emildimitrov**

### CODE REVIEW

File: concordances/cypher.go [hunk 1]
```diff
@@ -124,11 +126,10 @@ func (pcw CypherDriver) ReadByAuthority(authority string, identifierValues []str
...
 			Statement: `
-		MATCH (p:Concept)
-		WHERE exists(p.iso31661)
-		AND p.iso31661 IN {authorityValue}
-		MATCH (p)-[:EQUIVALENT_TO]->(canonical:Concept)
-		RETURN DISTINCT canonical.prefUUID AS canonicalUUID, labels(canonical) AS types, p.uuid as UUID, 'ISO-3166-1' as authority, p.iso31661 as authorityValue
+		MATCH (canonical:Location)
+		WHERE canonical.iso31661 IN {authorityValue}
+		AND exists(canonical.prefUUID)
+		RETURN DISTINCT canonical.prefUUID AS canonicalUUID, labels(canonical) AS types, canonical.uuid as UUID, 'ISO-3166-1' as authority, canonical.iso31661 as authorityValue
```
Comment (karsov) [inline concordances/cypher.go]: I don't see integration tests about this?

Comment (karsov) [reply concordances/cypher.go]: Isn't it better then to wait for the concepts-rw-neo4j service to be modified before merging this into master?

Comment (emildimitrov) [reply concordances/cypher.go]: Well in the scope of this increment concept-rw is irrelevant. This PR is more or less fix to what is already in prod

Comment (karsov) [reply concordances/cypher.go]: Ah, that's true!

P.S. I would suggest better branch naming and commit message the next time.

**Pull request merged**